Source/JavaScriptCore/inspector/InjectedScriptSource.js

```diff
@@ -1683,6 +1683,8 @@ CommandLineAPI.methods["timeStamp"] = function() { return inspectedGlobalObject.
 CommandLineAPI.methods["trace"] = function() { return inspectedGlobalObject.console.trace(...createIterableWithoutPrototypeFromArguments(arguments)); };
 CommandLineAPI.methods["warn"] = function() { return inspectedGlobalObject.console.warn(...createIterableWithoutPrototypeFromArguments(arguments)); };
 
+CommandLineAPI.methods["gatherRTCLogs"] = function() { return inspectedGlobalObject.@gatherRTCLogs(...createIterableWithoutPrototypeFromArguments(arguments)); };
```

Comment:
you may be better off moving this to [Source/WebCore/inspector/CommandLineAPIModuleSource.js](https://github.com/WebKit/WebKit/blob/main/Source/WebCore/inspector/CommandLineAPIModuleSource.js) as then it will only exist for pages (i.e. not for things like `JSContext`) and using the `CommandLineAPIHost` to bridge between JS and C++